src/Pages/ReviewWrite/ReviewWrite.js

```diff
@@ -1,7 +1,87 @@
-import React from 'react';
+import React, { useState } from 'react';
+import GrayMelon from './GrayMelon';
+import GreenMelon from './GreenMelon';
+import './ReviewWrite.scss';
 
 const ReviewWrite = () => {
-  return <div>reslissdfsdfsdfsdfst</div>;
+  const [input, setInput] = useState(10);
+  const [filed, setFiled] = useState([]);
+  // const reviewForm = document.querySelector('.ReviewWritePage');
+  // console.log(new FormData(reviewForm));
+  return (
+    <form className="ReviewWritePage">
```

Comment:
```suggestion
    <form className="reviewWrite">
```
최상위 부모 태그의 클래스 네임은 컴포넌트의 이름과 동일하게 작성해야 하고
Pascal Case는 컴포넌트 이름에만 작성하는 부분입니다.
className에는 camel Case로 작성해주세요!

아래 클래스네임을 보니 전부 파스칼 케이스로 작성 되어있습니다.
이 부분 카멜케이스로 변경해주세요!